packages/ui-extensions-react/src/surfaces/checkout/components.ts

```diff
@@ -73,6 +73,9 @@ export type {ListProps} from './components/List/List';
 export {ListItem} from './components/ListItem/ListItem';
 export type {ListItemProps} from './components/ListItem/ListItem';
 
+export {LoginWithShop} from './components/LoginWithShop/LoginWithShop';
+export type {LoginWithShopProps} from './components/LoginWithShop/LoginWithShop';
+
```

Comment:
This is a new component. We should add @mikewilson01 as a reviewer to ensure they are aware that will be discoverable in the new package and usable, even if the feature or documentation isn't ready yet.

Reply:
Hi @jamesvidler - thanks for pointing this out, yes this is safe to ship. It's purposely not (publicly) documented as it's intended to be used for a single merchant. It's currently gated behind a beta flag, so though it's discoverable, it's not usable.

Reply:
Great 👍